cupyx/scipy/fft/fft.py

```diff
@@ -0,0 +1,418 @@
+import cupy
+from cupy.cuda import cufft
+from cupy.fft.fft import (_fft, _default_fft_func, hfft as _hfft,
+                          ihfft as _ihfft)
+from cupy.fft.fft import fftshift, ifftshift, fftfreq, rfftfreq
+import numpy as np
+from numbers import Number
+
+
+__all__ = ['fft', 'ifft', 'fft2', 'ifft2', 'fftn', 'ifftn',
+           'rfft', 'irfft', 'rfft2', 'irfft2', 'rfftn', 'irfftn',
+           'hfft', 'ihfft',
+           'fftshift', 'ifftshift', 'fftfreq', 'rfftfreq']
+
+try:
+    import scipy.fft as _scipy_fft
+except ImportError:
+    class _DummyModule:
+        def __getattr__(self, name):
+            return None
+
+    _scipy_fft = _DummyModule()
+
+# Backend support for scipy.fft
+
+__ua_domain__ = 'numpy.scipy.fft'
+_implemented = {}
+
+
+def __ua_convert__(dispatchables, coerce):
+    if coerce:
+        try:
+            replaced = [
+                cupy.asarray(d.value) if d.coercible and d.type == np.ndarray
```

Comment:
Is there a reason this is `==` instead of `is`?

Reply:
Not really, I think both are equivalent here. Or do you think I should prefer `is`?

Reply:
They are equivalent, but in general I've seen type comparisons done with either `is` or `issubclass`.

Reply:
Unless there's some weird metaclass stuff going on in `d.type`. Which isn't the case here.

Reply:
I'd prefer `is` or `isinstance` to `==`.